sdk/metricsadvisor/azure-ai-metricsadvisor/README.md

```diff
@@ -15,15 +15,15 @@ Metrics Advisor is a scalable real-time time series monitoring, alerting, and ro
 
 ## _Disclaimer_
```

Comment:
you should remove everything except the disclaimer and this line at the top:
`# Microsoft Azure SDK for Python`

Reply:
ah I didn't realize it was remove everything my bad, I will fix :)

Reply:
and I see where it said that in the doc too oof

Reply:
no worries, I just updated it actually to make it clearer!